build/lib/modernizr.min.js

```diff
@@ -1 +1,313 @@
\ No newline at end of file
+/*!
+ * Modernizr v2.8.3
```

Comment:
@tahb It's minified now, I added another task to gruntfile in https://github.com/dxw/govsite/pull/39/commits/d002de149d5838af9151c75c1906e4fc400b6297.